docs/source/install.rst

```diff
@@ -414,6 +414,12 @@ All cached entries can be viewed using ``cmake -LAH`` from a build directory.
     Path to an existing directory used to cache :file:`proj.db` to speed-up
     subsequent builds without modifications to source SQL files.
 
+.. option:: EMBED_PROJ_DATA_PATH
+
+    .. versionadded:: 9.4
```

Comment:
9 4.1 is already released. Should it be 9.5, the next release?

Reply:
Yes

Reply:
Also, I think the description below this needs a bit more work. When I'm reading this it only clear to my that this hardcodes the `PROJ_DATA_PATH` into the binaries because I've read the discussion in this PR. A short explanation of why you might want to turn this off would be very helpful

Reply:
Hi, I have made the changes, please review this PR and if you have any questions, please let me know.